services/application/src/actions/user/update-custom-boolean-answers.js

```diff
@@ -21,9 +21,24 @@ module.exports = async ({
   // do not update user answers when passed answers are not an array
   if (!isArray(answers)) return user;
 
-  const toSet = answers
-    // .filter(({ value }) => optionIds.length) // ignore/unset fields without options
-    .map(({ fieldId, value }) => ({ _id: fieldId, value }));
+  // get all current answers as object { id, value }
+  const userObj = user.customBooleanFieldAnswers.reduce(
+    (obj, item) => Object.assign(obj, { [item._id]: item.value }), {},
```

Comment:
This should use rest spread (same as assign but cleaner)

`(obj, item) => ({ ...obj, [item._id]: item.value }), {},`